src/components/memorygame/MemoryGame/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { Col, Row } from "antd";
+import Confetti from "react-dom-confetti";
+import { Redirect } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import MemoryGameFactList from "../MemoryGameFactList/index";
+import "./index.css";
+import confettiConfig from "../confettiConfig";
+import MemoryGameCards from "../MemoryGameCards";
+
+const MemoryGame = () => {
+  const [t] = useTranslation();
+  const factsTitle = t("memoryGame.facts");
+  const gameTitle = t("memoryGame.title");
+  const [completed, setCompleted] = useState(false);
+  const [redirect] = useState();
+  const [matchedCards, setMatchedCards] = useState([]);
+
+  return redirect ? (
+    <>
+      <Redirect to="/" />
+    </>
+  ) : (
+    <Row className="memoryGame" justify="space-around">
+      <Confetti active={completed} config={confettiConfig} />
```

Comment:
If it is possible, it would be nice that you add a short note about what Confetti does in the component.

Reply:
I will add. Thank you. I'm waiting for other PR.